lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -349,8 +349,9 @@ protected function readProject($key, Repository $rep)
 
     public function getQgisPath()
     {
-        if (!$this->file) {
-            $this->file = realpath($this->repository->getPath()).'/'.$this->key.'.qgs';
+        $path = $this->repository->getPath();
+        if (!$this->file && $path != '' && $path != false) {
+            $this->file = realpath($path).'/'.$this->key.'.qgs';
```

Comment:
the `realpath()` should be done into the Repository class, so the code calling `getPath()` don't have to do it